ui/src/app/subnets/views/VLANDetails/VLANSubnets/VLANSubnets.test.tsx

```diff
@@ -0,0 +1,61 @@
+import { render, screen, within } from "@testing-library/react";
+import { Provider } from "react-redux";
+import { MemoryRouter } from "react-router-dom";
+import configureStore from "redux-mock-store";
+
+import VLANSubnets from "./VLANSubnets";
+
+import subnetsURLs from "app/subnets/urls";
+import {
+  rootState as rootStateFactory,
+  subnet as subnetFactory,
+  subnetState as subnetStateFactory,
+  subnetStatistics as subnetStatisticsFactory,
+  vlan as vlanFactory,
+  vlanState as vlanStateFactory,
+} from "testing/factories";
+
+const mockStore = configureStore();
+
+it("renders correct details", () => {
```

Comment:
It feels like this doesn't really test what kind of UI we're showing to the user. I'm not totally sure what the solution is, we don't need to test the internals of MainTable, but these tests would pass even if it was displaying a couple of `<p>`s on the page.

Reply:
I could increase the specificity to assert the details are indeed inside a table, instead of just floating around somewhere on the page. TBH I'm still trying to work my way around how we're "supposed" to be writing tests with RTL.